docs/source/standard/item.rst

```diff
@@ -18,7 +18,7 @@ Schema
     string, multilingual, required
 
     |ocdsDescription|
-    A description of the goods, services to be provided.
```

Comment:
For the fields to be described _ocds description_ is the best variant to be used.
So either use the descriptions provided by OCP (without changing the content) or remove |ocdsDescription|